pytest.ini

```diff
@@ -34,11 +34,11 @@ testpaths =
     tests/integration/
 env =
     # a weird timezone to check that tests aren't assuming the local timezone
-    TZ=XXX-09:35
+    # TZ=XXX-09:35
```

Comment:
This has been removed on master, setting TZ within a Python session doesn't work right.